backend/tracim_backend/lib/core/mention.py

```diff
@@ -112,6 +114,41 @@ def get_mentions_from_html(cls, content_id: int, html: str) -> typing.List[Menti
             )
         return mentions
 
+    @classmethod
+    def get_email_html_from_html_with_mention_tags(
+        cls, session: TracimSession, cfg: CFG, translator: Translator, html: str
+    ) -> str:
+        """
+        This method will replace every mention tag by a simple string mention.
+        :param session: session to use for database access
+        :param cfg: current config
+        :param translator: translator to use for role translation
+        :param html: html to parse
+        :return: html with mention tags replaced by simple string mention
+
+        Example:\n
+        html = <div>Hello <html-mention userid="1"></html-mention>!</div>\n
+        return = <div>Hello @foo!</div>\n
+        Where @foo is the username of user with id 1
```

Comment:
```suggestion
        Example:
        ```
        html = <div>Hello <html-mention userid="1"></html-mention>!</div>
        return = <div>Hello @foo!</div>
        ```
        Where @foo is the username of user with id 1
```

?